languages/swift/exercises/concept/strings-and-characters/.docs/introduction.md

```diff
@@ -0,0 +1,111 @@
+Strings in Swift are a collection of characters, where characters are, more or less, a single printable element. So strings in Swift are able to contain Unicode characters and emoji.
+
+Strings are easily created in Swift through string literals. String literals are similar to those seen in many other programming languages, characters enclosed between a pair of double quotation marks (`"`). String literals can include the following special characters:
+\\0 (null character), \\\\ (backslash), \\t (horizontal tab), \\n (newline), \\r (carriage return), \\" (double quotation mark) and \\' (single quotation mark).
```

Comment:
```suggestion
`\0` (null character), `\\` (backslash), `\t` (horizontal tab), `\n` (newline), `\r` (carriage return), `\"` (double quotation mark) and `\'` (single quotation mark).
```

Reply:
The introduction.md document doesn't have to be exhaustive, it should give the student just enough context to figure out the solution themselves. As such, you could consider shortening this a bit by mentioning that certain characters require escaping by prefixing them with the `\` character, and then naming one example. The after.md can then be exhaustive.